core/transaction.go

```diff
@@ -71,7 +72,7 @@ type TransactionReceipt struct {
 
 type Transaction interface {
 	// Todo: Add Hash as a field to all the Transaction Objects
-	Hash() *felt.Felt
+	Hash(*utils.Network) *felt.Felt
```

Comment:
Network is just a `uint8` use the value instead of a pointer please.

Reply:
Done